src/code/z_sram.c

```diff
@@ -136,9 +136,52 @@ static ItemEquips sNewSaveEquips = {
 };
 
 static Inventory sNewSaveInventory = {
-    { 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF,
-      0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF }, // items
-    { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 },                         // ammo
+    // items
+    {
+        0xFF, // SLOT_STICK
+        0xFF, // SLOT_NUT
+        0xFF, // SLOT_BOMB
```

Comment:
```suggestion
    // items
    {
        ITEM_NONE, // SLOT_STICK
        ITEM_NONE, // SLOT_NUT
        ITEM_NONE, // SLOT_BOMB
```
Looks like every entry here could use this enum

Reply:
I didn't do it because it's strictly speaking out of scope of the PR (I am OrGaNiZeD), but yeah there's no harm doing it now I guess